branchtesting/src/main/java/branchexample/App.java

```diff
@@ -10,7 +10,8 @@ public void aMethod( String[] args )
     		// TODO fix this too..
     		int j = 5 / i;
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Make sure "i" can't be zero before doing this division. ([squid:S3518](http://localhost:9721/coding_rules#rule_key=squid%3AS3518))

[See it in SonarQube](http://localhost:9721/project/issues?id=com.example.bcipollone%3Abranchtesting&pullRequest=2&issues=AWU6UGSVcMk10FtshGcy&open=AWU6UGSVcMk10FtshGcy)

[ISSUE_KEY]: AWU6UGSVcMk10FtshGcy